modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -1,29 +1,22 @@
 import { Arr, Cell, Option, Thunk } from '@ephox/katamari';
-import { TableLookup } from '@ephox/snooker';
+import { TableLookup, RunOperation } from '@ephox/snooker';
 import { Element, Node } from '@ephox/sugar';
 import Editor from 'tinymce/core/api/Editor';
 import * as TableTargets from '../queries/TableTargets';
 import { Selections } from './Selections';
 import * as TableSelection from './TableSelection';
 
-export interface Targets {
```

Comment:
Moved this to RunOperations as CombinedTargets, since that's where all the other target-related interfaces are